addons/mail/static/src/components/message/message.xml

```diff
@@ -77,11 +83,14 @@
                                     Anonymous
                                 </div>
                             </t>
-                            <t t-if="message.date">
+                            <t t-if="!message.hasSendError and message.date">
```

Comment:
Shouldn't message pending or in error have no date now?

Reply:
This PR is quite old now, I think that change was made after.